### PR TITLE
Adjusts the age minimum and maximum to 18 and 100 respectively

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Adjusts the age minimum and maximum to 18 and 100 respectively.

## Why It's Good For The Game

Why can't my spacemen be a one hundred year old decrepit ghoul? It's restricting my roleplay.

## Changelog

:cl:
qol: Adjusts the age minimum and maximum to 18 and 100 respectively
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -403,8 +403,8 @@
 #define OFFSET_NECK "neck"
 
 //MINOR TWEAKS/MISC
-#define AGE_MIN 17 //youngest a character can be
```
Comment (Fikou) [inline code/__DEFINES/mobs.dm:406]: Will preferences automatically update 17 year old characters to 18?

**Comment by AlinaStarkova** [comment]
the fact apprentice age is only 1 year less than a wizard seems pointless lol, would make more sense as 25, on their way but not "about to be a wizard anyway" lol

as for age, even today humans can live up to 123~ i assume in space with higher end treatment this would be a norm, so a max age of 125 probably makes more sense

that said i cant imagine a company hiring workers older than say 75, as thats unproductive and you generally gotta pay older people more, but its just flavor stuff anyway so i dont really care if it doesnt make quite as much sense from an economic standpoint

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -403,8 +403,8 @@
...
-#define AGE_MAX 85 //oldest a character can be
+#define AGE_MIN 18 //youngest a character can be
```
[the remark this answers is not in the document]
Comment (AlinaStarkova) [reply code/__DEFINES/mobs.dm:406]: same, makes sense on some of TGs downstreams cuz *cough* nsfw content, but tg doesnt have that sorta content so there really isnt any need for a character age to be 18 min specifically, honestly we could probably even go as low as maybe 15 (i would say even lower but then we risk people trying to roleplay as an annoying child)

**Comment by DrDiasyl** [comment]
No more 17 years old catgirls? BEST PR 👏👏👏👏

File: code/__DEFINES/mobs.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/__DEFINES/mobs.dm:406]: yes afaik

File: code/__DEFINES/mobs.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/__DEFINES/mobs.dm:406]: well this is an 18 plus codebase

File: code/__DEFINES/mobs.dm (hunk 1 above)
Comment (Fikou) [reply code/__DEFINES/mobs.dm:406]: please check so

**Comment by Mothblocks** [comment]
You already tried this with #59835 and it got closed, do you have an okay from oranges?

**Comment by Iamgoofball** [comment]
i forgot about that PR sorry
can we keep the age max at 100 though?

**Comment by Mothblocks** [comment]
yeah seems funny

**Comment by optimumtact** [comment]
piss off cunt

**Comment by SeeminglyTypicalUsername** [comment]
> piss off cunt

Who are you grooming, Oranges?

**Pull request closed without merging**